### ISSUE TITLE
How to add Social Media Links when creating customer through API

### ISSUE THREAD

**Issue by kirschkern** [freescout-helpdesk/freescout#3977]
I am using the following JSON to create a new conversation and also create a new customer. This works well except for the social media profile. Instead of beeing saved as "Linkedin", it will be saved as "Others". I've also tried several variations like "linkedin", LinkedIn" etc.

When looking into the database, the social media profile will be saved with type:12, while the type for a linkedin entry is 3. However, when using 3 as type, it is still saved as type:12.

Using the FreeScout template in Zapier, the social media profiles cannot be defined at all.

```
{
    "type": "email",
    "mailboxId": 4,
    "subject": "Karriere Kick Interessent über LinkedId",
    "customer": {
        "email": "daniel_4@kirschkern.de",
        "firstName": "Daniel",
        "lastName": "Kirsch",
        "company": "Test",
        "jobTitle": "CEO",
        "socialProfiles": [
            {
                "value": "https://www.linkedin.com/in/daniel-kirsch-0a9184138/",
                "type": "linkedin"
            }
        ]
    },
    "threads": [
        {
            "text": "Die Nachricht der Mail, ggf. mit weiteren Daten",
            "type": "customer",
            "customer": {
                "email": "daniel_4@kirschkern.de"
            }
        }
    ]
}
```

What do I need to set the type to, to save it as a "Linkedin" profile?

**Comment by freescout-helpdesk** [comment]
Fixed in API & Webhooks Module v1.0.78

**Comment by kirschkern** [comment]
> Fixed in API & Webhooks Module v1.0.78

Great! Fast as always!

May you tell me, what value to add for the "type" property?

**Comment by freescout-helpdesk** [comment]
```
"socialProfiles": [
            {
                "value": "https://www.linkedin.com/in/daniel-kirsch-0a9184138/",
                "type": "linkedin"
            }
        ]
```
